README.md

```diff
@@ -41,7 +41,7 @@ print(ot.get_association('ENSG00000157764-EFO_0005803')[0])
 
 e_for_target = ot.get_evidence_for_target('BRAF')
 for e in e_for_target:
-    print(result_to_json(e))
+    print(json.dumps(e))
```

Comment:
I do not know about all the responses, but the example response is a dict already and there was an error with the example given, this applies to only the example.